### PR TITLE
added `EnableExperimentalGossipEncryption`

### PR DESCRIPTION
**Pull request by thefactremains**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -955,6 +955,19 @@ Note that the gossip port and gossip protocol are used to determine cluster heal
 | This feature's ``config.json`` setting is ``"UseExperimentalGossip": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------------+
 
+Enable Expirimental Gossip Encryption
```
Comment (agnivade) [inline source/administration/config-settings.rst:958]: Experimental

Comment (justinegeffen) [inline source/administration/config-settings.rst:958]:
```suggestion
Enable Experimental Gossip Encryption
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -955,6 +955,19 @@ Note that the gossip port and gossip protocol are used to determine cluster heal
 | This feature's ``config.json`` setting is ``"UseExperimentalGossip": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------------+
 
+Enable Experimental Gossip Encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: All communication through the cluster using the gossip protocol will be encrypted
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:961]:
```suggestion
**True**: All communication through the cluster using the gossip protocol will be encrypted.
```

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -955,6 +955,19 @@ Note that the gossip port and gossip protocol are used to determine cluster heal
...
+
+**False**: All communication using gossip protocol remains unencrypted.
+
+The encryption uses AES-256 by default, and it is not kept configurable by design. However, if one wishes, they can set the value in Systems table manually for the ``ClusterEncryptionKey`` row. A key is a byte array converted to base64. It should be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:965]:
```suggestion
The encryption uses AES-256 by default, and it is not kept configurable by design. However, you can manually set the ``ClusterEncryptionKey`` row value in the Systems table. A key is a byte array converted to base64. It should be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256.
```

**Review by agnivade — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**